OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -46,3 +46,11 @@ As you don't have execution rights to it, however, you get the error `Operation
 Run the container with `-v /dev/shm --tmpfs /dev/shm:rw,exec,size=<yoursize>` instead, the important part being the `exec` in `--tmpfs /dev/shm:rw,exec,size=<yoursize>`.
 Also make sure you assign an appropriate size as the default Docker uses is only 64MB. Assigning 1GB or  more is recommended.
 
+## ORA-12637: Packet receive failed
+When initially connecting to your 19c (or higher) database the client may appear to hang and timeout after a few minutes with: `ORA-12637: Packet receive failed`
+
+Some network stacks do not correctly handle Out Of Band breaks which are enabled by default in Oracle Database 19c. Problems have been seen on _docker-engine-19.03.1.ol-1.0.0.el7_. You may disable this feature by setting `DISABLE_OOB=ON` in the clients _sqlnet.ora_ file. By default the Oracle Instant Client for Linux will use _/<instant_client_path>/network/admin/sqlnet.ora_, _$TNS_ADMIN/sqlnet.ora_ and _~/.sqlnet.ora_. For example you could use
```

Comment:
Still not quite there.  It's not always enabled. Some official doc which may help is https://www.oracle.com/pls/topic/lookup?ctx=dblatest&id=GUID-554C0311-68FB-4628-AC8D-C22D8ADDE995

Your later "disable this feature" will also need tweaking to match.

Technically it's Oracle Net 19c, not Oracle DB 19c, but I don't think getting too precise will help users here.